tests/unit/test_peak_collection.py

```diff
@@ -24,19 +45,38 @@ def test_background_enum():
     assert len(BackgroundFunction.getFunction('linear').native_parameters) == 2
 
 
-def test_peak_collection():
-    # create test data with two peaks - both peaks will be normalized gaussians
-    NUM_SUBRUN = 2
+def check_peak_collection(peak_shape, NUM_SUBRUN, target_error):
+
+    """check the peak collection
+
+    Parameters
+    ----------
+    subruns : numpy.array
+        1D numpy array for sub run numbers
+    peak_shape : str
+            Peak shape
+    target_errors : numpy.ndarray
+        numpy structured array for peak/background parameter fitted target error
+
+    Returns
+    -------
+
+    """
     subruns = np.arange(NUM_SUBRUN) + 1
     chisq = np.array([42., 43.])
-    raw_peaks_array = np.zeros(NUM_SUBRUN, dtype=get_parameter_dtype('Gaussian', 'Linear'))
-    raw_peaks_array['Height'] = [1, 2]
+    raw_peaks_array = np.zeros(NUM_SUBRUN, dtype=get_parameter_dtype(peak_shape, 'Linear'))
+    if peak_shape == 'PseudoVoigt':
+        raw_peaks_array['Intensity'] = [1, 2]
```

Comment:
You need to add in the mixing parameter as well. It needs to be between zero and one.